geographic_msgs/GeoPoint ll_point
---
geometry_msgs/Point map_point
